org.eclipse.xtext.xbase.lib/src/org/eclipse/xtext/xbase/lib/Conversions.java

```diff
@@ -42,16 +44,142 @@ private Conversions() {
 	}
 
 	/**
-	 * Wraps {@code object} in a list if and only if {@code object} is an array. Works for primitive and
-	 * object-component types.
+	 * Wraps {@code object} in a list if and only if {@code object} is an array.
```

Comment:
Please try to avoid unrelated changes to the formatting. It makes digesting the PR harder than strictly necessary.